templates/ding-item-list_item.tpl.php

```diff
@@ -13,14 +13,9 @@
   </div>
   <div class="item-details">
     <div class="item-title"><a href="/ting/object/<?php print $faust; ?>"><?php print $title; ?></a></div>
-    <div class="item-author"><?php if (!empty($author)) { print t('By @author', array('@author' => $author)); } ?> (<?php print $year;?>)</div>
-    <?php if ($has_rating): ?>
```

Comment:
Why is rating removed?

Reply:
![image](https://user-images.githubusercontent.com/3027336/47795094-808eb080-dd2a-11e8-8079-1e65ab8366cb.png)
Because it is not displayed on the production widget

Reply:
I definitely think it should implement ratings (from p2), like carousel does.